packages/flutter_tools/test/general.shard/github_template_test.dart

```diff
@@ -195,3 +310,10 @@ class SuccessShortenURLFakeHttpClient extends FakeHttpClient {
     return SuccessFakeHttpClientRequest();
   }
 }
+
+class FakeError implements Error {
+  @override
+  StackTrace get stackTrace => StackTrace.fromString('''
+#0      _File.open.<anonymous closure> (dart:io/file_impl.dart:366:9)
+#1      _rootRunUnary (dart:async/zone.dart:1141:38)''');
+}
```

Comment:
would be good to have a test that pretends to have sensitive information in an unknown exception class (like FakeError with a toString that contains fake PII), and verifies that none of that information leaks out.

Reply:
Added in https://github.com/flutter/flutter/pull/96250/files#diff-c35c1425f1f396a35eca11a3edfae52343a312256d6df2cea65d238867cc97ffR313.